frontends/mit-open/src/pages/FieldPage/FieldSearch.tsx

```diff
@@ -128,19 +175,40 @@ const FieldSearch: React.FC<FeildSearchProps> = ({
   const page = +(searchParams.get("page") ?? "1")
 
   return (
-    <SearchDisplay
-      page={page}
-      requestParams={params}
-      setPage={setPage}
-      facetManifest={facetManifest}
-      facetNames={facetNames}
-      constantSearchParams={constantSearchParams}
-      hasFacets={hasFacets}
-      setParamValue={setParamValue}
-      clearAllFacets={clearAllFacets}
-      toggleParamValue={toggleParamValue}
-      patchParams={patchParams}
-    />
+    <div>
+      <SearchInputContainer>
+        <StyledSearchInput
+          value={currentText}
+          onChange={(e) => setCurrentText(e.target.value)}
+          onSubmit={(e) => {
+            setCurrentTextAndQuery(e.target.value)
+          }}
+          onClear={() => {
+            setCurrentTextAndQuery("")
+          }}
+          classNameInput="input-field"
+          classNameSearch="button-field"
+          placeholder="Search for courses, programs, and learning materials..."
+        />
+      </SearchInputContainer>
+
+      <SearchDisplay
+        page={page}
+        requestParams={params}
+        setPage={setPage}
+        facetManifest={facetManifest}
+        facetNames={facetNames}
+        constantSearchParams={constantSearchParams}
+        hasFacets={hasFacets}
+        setParamValue={setParamValue}
+        clearAllFacets={clearAllFacets}
+        toggleParamValue={toggleParamValue}
+        patchParams={patchParams}
+        showProfessionalToggle={
```

Comment:
This is maybe out of scope for this PR, but I'm a bit surprised to find that we don't have any tests for `SearchDisplay`. It would be good to add some tests for this component as it has a lot of arguments that need to be evaluated. I think it's okay to make an issue to add tests for it and merge this PR without them, but things like the "professional toggle is hidden when this argument is passed" should be easy to assert.